test/e2e/data/e2e_conf.yaml

```diff
@@ -116,6 +136,7 @@ variables:
   OPENSTACK_VOLUME_TYPE_ALT: "test-volume-type"
   CONFORMANCE_WORKER_MACHINE_COUNT: "5"
   CONFORMANCE_CONTROL_PLANE_MACHINE_COUNT: "1"
+  INIT_WITH_KUBERNETES_VERSION: "v1.25.0"
```

Comment:
This is for the [secondary management cluster](https://github.com/kubernetes-sigs/cluster-api/blob/e032bec7c04c45b42d92b7bb9bd4f16765218181/test/e2e/clusterctl_upgrade.go#L73). It will be used when rendering the cluster-template [here](https://github.com/kubernetes-sigs/cluster-api/blob/e032bec7c04c45b42d92b7bb9bd4f16765218181/test/e2e/clusterctl_upgrade.go#L196-L209).
I also noticed that the installation image is stuck on 1.18, but we use the CI artifacts script to actually upgrade before kubeadm is run.